python/paddle/nn/layer/loss.py

```diff
@@ -62,6 +63,8 @@ class CrossEntropyLoss(fluid.dygraph.Layer):
             If :attr:`size_average` is ``'sum'``, the reduced sum loss is returned.
             If :attr:`reduction` is ``'none'``, the unreduced loss is returned.
             Default is ``'mean'``.
+        ignore_index (int64, optional): Specifies a target value that is ignored
```

Comment:
Add shape description for `input` and `label` in line 53 and line 55. If shape is more than 2D, also add unit testing for them.

Reply:
Thanks, DONE